ships/0038-release-branching-backports.md

```diff
@@ -127,78 +129,260 @@ such situations.
+- `operator`: The `release.yml` workflow will receive a new required parameter,
+  `release-branch`. This will be passed to the standard git checkout action.
+- `cli`: The `cli` repository will add a new workflow, `release-tag.yml`, which will
+  create the tag for the upcoming release. This will accept `release-branch` as
+  required parameter, to be passed to the standard git checkout action:
+
+  ```yaml
+  name: Create Release Tag
+  on:
+    workflow_dispatch:
+      inputs:
+        release-branch:
+          required: true
+          type: string
+          description: "Branch to use for creating the release tag"
+        version-tag:
+          required: true
+          type: string
+          description: "Full semantic version of the release (in vX.Y.Z format)"
+  jobs:
+    create-release-tag:
+      ... # permissions, check out ${{ inputs.release-branch }}
+      - name: Tag release
+        env:
+          - VERSION_TAG: ${{ inputs.version-tag }}
+        run: |
+          git tag ${VERSION_TAG}
+          git push --tags
+  ```
+
+  Once the tag is created, the `cli` release workflow should work as-is.
+
+#### GitHub Branch Protection
+
+Each repository will need to create a new branch protection rule that applies to
+`release-v*` branches. Branch protection settings from the `main` branch will need to
+be manually copied over to the release branch rule.
+This is due to simplifying limitations in how GitHub does pattern matching for
+[branch protection rules](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-protected-branches/managing-a-branch-protection-rule#about-branch-protection-rules).
+
+The branch protection policy needs to allow GitHub actions to push branches and tags.
+
+#### Backport Process
+
+Backporting can be done in one of two ways:
+
+1. "Manually": this would involve a contributor running `git cherry-pick` on their
+   local machine against a release branch, resolving any merge conflicts, then
+   opening a pull request
+2. Prow `cherrypick` bot: our existing configuration for OpenShift CI will be
+   updated to enable the Prow cherrypick plugin. The bot can automatically create
+   cherrypick pull requests by adding a `/cherrypick <branch-name>` comment to an
+   existing pull request, either when it is open or after merge. See the Prow
+   docs [1] for more details.
+
+[1] https://docs.prow.k8s.io/docs/components/external-plugins/cherrypicker/
 
-No need to outline all of the test cases, just the general strategy. Anything that would count as
-tricky in the implementation and anything particularly challenging to test should be called out.
+### Test Plan
 
-All code is expected to have adequate tests (eventually with coverage expectations).
-```
+CI jobs for each component repository will need to support execution from a branch
+that matches the release branch pattern. This is done by updating the `on` stanza of
```

Comment:
Hm, I was about to suggest this, but wonder why we run the actions like Verify on push at all.

```suggestion
that matches the release branch pattern. This is done by updating the `on.pull_request` stanza of
```